### PR TITLE
changes to docs to bring up to date

### PR DESCRIPTION
**Pull request by cdolfi**

### CODE REVIEW

File: docs/new_viz_guidance.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ individual visualizations are imported and further organized below.
 
 ### Visualization Template
 
-[Visualization Template](https://github.com/oss-aspen/8Knot/blob/dev/pages/visualization_template/viz_template.py) gives you the outline to making your next visualization.
+The [Visualization Template](https://github.com/oss-aspen/8Knot/blob/dev/pages/visualization_template/viz_template.py) can take you step by step on creating your new visualization with boilerplate code with the overall structure and needs of a visualization.
```
Comment (JamesKunstle) [inline docs/new_viz_guidance.md:28]: "takes you step-by-step through the process of creating a new visualization." 

The original sentence is a run-on. I think it's sufficient to point to the guide and say "Here's where you learn how to do the thing".

File: docs/new_viz_guidance.md [hunk 2]
```diff
@@ -35,14 +35,13 @@ Database are in the folder /queries/. These queries may be reused across multipl
 visualizations. When considering creating your own visualization, please
 see whether the available queries to our database instance are sufficient.
 
-If the existing queries won't suffice, the most straight-forward way to create
-your own is to copy one of the existing queries into the same directory with a
-descriptive filename:
+If the existing queries do not suffice:
+1. See if there is some minor changes or additional columns that could be added to
```
Comment (JamesKunstle) [inline docs/new_viz_guidance.md:39]: "Exhaust the possibility of making minor changes, such adding new columns or aggregations, to existing queries."

File: docs/new_viz_guidance.md [hunk 3]
```diff
@@ -35,14 +35,13 @@ Database are in the folder /queries/. These queries may be reused across multipl
...
+an existing query to get the necessary data.
+2. If a completely new query is neccessary, use the [Query Template](https://github.com/oss-aspen/8Knot/blob/dev/queries/query_template.py)
+to build a new query.
 
-e.g. "mv commits\_query.py ./comments\_query.py"
-
-Then, change as few things as possible in the newly created file to meet the
-needs of your new query. Make sure to include comments.
+Note: make sure to do a "docker system prune -af" before rebuilding to be able to access any new or edited queries.
```
Comment (JamesKunstle) [inline docs/new_viz_guidance.md:44]: Using 'docker compose down' is sufficient. 'docker system prune -af' will destroy EVERYTHING they're running, including any other docker services or volumes.

**Review by JamesKunstle — CHANGES_REQUESTED** [review]
A few wording suggestions and a docker command edit.

**Review by JamesKunstle — APPROVED** [review]
Looks perfect

**Comment by JamesKunstle** [comment]
@cdolfi Could you please run the spell checker once more locally? Then it's good to merge.

**Comment by cdolfi** [comment]
@JamesKunstle How do I go about doing it? From what I can see in my local check the spell check is not in there

**Comment by JamesKunstle** [comment]
> @JamesKunstle How do I go about doing it? From what I can see in my local check the spell check is not in there

Ah yeah, I see, that's a Github action. Just correct the misspelling that's noted in the action.

**Pull request merged**